daal4py/sklearn/cluster/_k_means_0_23.py

```diff
@@ -108,7 +109,11 @@ def is_string(s, target_str):
         kmeans_init_res = kmeans_init.compute(X)
         centroids_ = kmeans_init_res.centroids
     else:
-        raise ValueError("Cluster centers should either be 'k-means++', 'random', 'deterministic' or an array")
+        raise ValueError(
+                f"init should be either 'k-means++', 'random', a ndarray or a "
+                f"callable, got '{cluster_centers_0}' instead.")
+    if verbose:
+        print("Initialization complete")
```

Comment:
Logging in stock sklearn make with help `print` ?

Reply:
yes, see https://github.com/scikit-learn/scikit-learn/blob/1ac9d6817da4d54d07f11e538d3447290cedfca3/sklearn/cluster/_kmeans.py#L990

Reply:
Ok, thanks